spec/controllers/application_drafts_controller_spec.rb

```diff
@@ -168,7 +168,7 @@
 
       context 'for a valid draft' do
         before do
-          allow_any_instance_of(ApplicationDraft).to receive(:valid?).with(:apply).and_return(true)
+          allow_any_instance_of(ApplicationDraft).to receive(:ready?).and_return(true)
```

Comment:
The tests should pass with the old stub in place since `#ready?` only forwards to that.